modules/nextflow/src/test/groovy/nextflow/executor/BashFunLibTest.groovy

```diff
@@ -88,8 +92,8 @@ class BashFunLibTest extends Specification {
 
         """
         cmds=()
-        cmds+=("true")
         cmds+=("false")
+        cmds+=("true")
```

Comment:
Is there a real reason for this change? I'd say no?

Reply:
Yes, there is.  With the previous/original order, the test case does not catch failing tasks unless they're at the end of the list.  This test exercises the more general case.
This change, in a separate test script, is what allowed me to track down the errant behavior.